### PR TITLE
Add web-name to third party list

### PR DESCRIPTION
**Pull request by vimalverma558**
Simple module to generate random usernames

### CODE REVIEW

File: database.json [hunk 1]
```diff
@@ -3079,6 +3079,12 @@
     "repo": "deno-watch",
     "desc": "A pure deno file watcher."
   },
+  "web-name": {
+    "type": "github",
+    "owner": "letskhabar",
+    "repo": "web-name",
+    "desc": "simple module to generate random usernames"
+  },
```
Comment (alirealasad) [inline database.json:3087]: database names with dashes are not allowed so can change the name to web_name, webName something else

Comment (vimalverma558) [reply database.json:3087]: Thanks for suggestions 😊😘

Comment (s1mpson) [reply database.json]: > database names with dashes are not allowed so can change the name to web_name, webName something else

@alirealasad Are module names case-sensitive? That would be a disaster having different modules named `webname`, `webName`, `Webname`, `wEbname` and so on

Comment (vimalverma558) [reply database.json]: Web_name is also a option

**Pull request merged**